app/stac_api/models.py

```diff
@@ -46,8 +46,11 @@
 DEFAULT_SUMMARIES_VALUE = {"eo:gsd": [], "geoadmin:variant": [], "proj:epsg": []}
 
 
-def get_default_stac_extensions():
-    return list(DEFAULT_STAC_EXTENSIONS.values())
+def get_default_stac_extensions(is_collection=False):
+    if is_collection: # pylint: disable=no-else-return
+        return list()
+    else:
+        return list(DEFAULT_STAC_EXTENSIONS.values())
```

Comment:
Don't disable pylint here simply write code accordingly to it:
```
if is_collection:
    return list()
return list(DEFAULT_STAC_EXTENSIONS.values())
```

Reply:
Makes sense :-)